File: backend/DataAccess/DaoManual/システム共有/tt_affilterDto.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: DTO定義
//             詳細条件設定テーブル
// 作成日　　: 2023.01.23
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tt_affilterDto : DaEntityModelBase
    {
        public const string TABLE_NAME = "tt_affilter";
        public string kinoid { get; set; }                        //機能ID
        public Enum詳細検索条件区分 jyokbn { get; set; }          //条件区分
        public int jyoseq { get; set; }                           //条件シーケンス
        public string hyojinm { get; set; }                       //詳細条件表示名
        public Enumコントローラータイプ ctrltype { get; set; }    //コントローラータイプ
        public bool hyojiflg { get; set; }                        //表示フラグ
        public int? sort { get; set; }                            //並び順
        public string? placeholder { get; set; }                  //入力説明
        public bool rangeflg { get; set; }                        //範囲フラグ
        public string sethanyokbn1 { get; set; }                  //設定用汎用区分1
        public string? sethanyokbn2 { get; set; }                 //設定用汎用区分2
        public string? sethanyokbn3 { get; set; }                 //設定用汎用区分3
        public string? sethanyokbn4 { get; set; }                 //設定用汎用区分4
        public string? sethanyokbn5 { get; set; }                 //設定用汎用区分5
        public Enum取得元区分 getkbn { get; set; }                //検索対象データ取得区分
        public string tablenm1 { get; set; }                      //テーブル物理名1
        public string komokunm1 { get; set; }                     //項目物理名1
        public string? keycd1 { get; set; }                       //主キーコード1
        public string? itemcd1 { get; set; }                      //項目コード1
        public string? tablenm2 { get; set; }                     //テーブル物理名2
        public string? komokunm2 { get; set; }                    //項目物理名2
        public string? keycd2 { get; set; }                       //主キーコード2
        public string? itemcd2 { get; set; }                      //項目コード2
        public string? tablenm3 { get; set; }                     //テーブル物理名3
        public string? komokunm3 { get; set; }                    //項目物理名3
        public string? keycd3 { get; set; }                       //主キーコード3
        public string? itemcd3 { get; set; }                      //項目コード3
    }
}

File: backend/JbdGjsService/Db/DaoManual/システム共有/tt_affilterDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             詳細条件設定テーブル
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Db
    Public Class tt_affilterDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tt_affilter"
        Public Property kinoid As String                        '機能ID
        Public Property jyokbn As Enum詳細検索条件区分          '条件区分
        Public Property jyoseq As Integer                           '条件シーケンス
        Public Property hyojinm As String                       '詳細条件表示名
        Public Property ctrltype As Enumコントローラータイプ    'コントローラータイプ
        Public Property hyojiflg As Boolean                        '表示フラグ
        Public Property sort As Integer                            '並び順
        Public Property placeholder As String                  '入力説明
        Public Property rangeflg As Boolean                        '範囲フラグ
        Public Property sethanyokbn1 As String                  '設定用汎用区分1
        Public Property sethanyokbn2 As String                 '設定用汎用区分2
        Public Property sethanyokbn3 As String                 '設定用汎用区分3
        Public Property sethanyokbn4 As String                 '設定用汎用区分4
        Public Property sethanyokbn5 As String                 '設定用汎用区分5
        Public Property getkbn As Enum取得元区分                '検索対象データ取得区分
        Public Property tablenm1 As String                      'テーブル物理名1
        Public Property komokunm1 As String                     '項目物理名1
        Public Property keycd1 As String                       '主キーコード1
        Public Property itemcd1 As String                      '項目コード1
        Public Property tablenm2 As String                     'テーブル物理名2
        Public Property komokunm2 As String                    '項目物理名2
        Public Property keycd2 As String                       '主キーコード2
        Public Property itemcd2 As String                      '項目コード2
        Public Property tablenm3 As String                     'テーブル物理名3
        Public Property komokunm3 As String                    '項目物理名3
        Public Property keycd3 As String                       '主キーコード3
        Public Property itemcd3 As String                      '項目コード3
    End Class
End Namespace
